src/scripts/extensions/authenticationHelper.ts

```diff
@@ -139,4 +143,43 @@ export class AuthenticationHelper {
 		// Note that we are returning true by default to ensure the N-1 scenario.
 		return userInfo.cookieInRequest !== undefined ? userInfo.cookieInRequest : true;
 	}
+
+	/**
+	 * fetch the user data bounday from the emailAddress
+	 * @param userInfo 
+	 * @returns user data boudary
+	 */
+	private async getUserDataBoundary(userInfo: UserInfoData): Promise<string | undefined> {
+		try {
+			if (!userInfo) {
+				return undefined;
+			}
+			if (userInfo.authType === AuthType[AuthType.Msa]) {
+				return DataBoundary[DataBoundary.GLOBAL];
+			}
+			let domainValue;
+			if (!userInfo.emailAddress) {
+				return undefined;
+			} else {
```

Comment:
Else not required since we are returning from if.

Reply:
Resolved this